docs/input_output/SUMMA_input.md

```diff
@@ -10,62 +10,94 @@ SUMMA input files are either ASCII format or NetCDF. The general characteristics
-
-1. `META_TYPE`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `META_FORCE`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `META_LOCALPARAM`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `OUTPUT_CONTROL`: File path for the [output control file](#infile_output_control) (relative to `SETNGS_PATH`).
-
-1. `META_LOCALINDEX`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `META_BASINPARAM`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `META_BASINMVAR`: No longer used as of SUMMA 2.x - simply specify `'N/A'`.
-
-1. `LOCAL_ATTRIBUTES`: File path for the [local attributes file](#infile_local_attributes) (relative to `SETNGS_PATH`).
-
-1. `LOCALPARAM_INFO`: File path for the [local parameters file](#infile_local_parameters) (relative to `SETNGS_PATH`).
-
-1. `BASINPARAM_INFO`: File path for the [basin parameters file](#infile_basin_parameters) (relative to `SETNGS_PATH`).
-
-1. `FORCING_FILELIST`: File path for the [list of forcing files file](#infile_forcing_list) (relative to `SETNGS_PATH`).
-
-1. `MODEL_INITCOND`: File path for the [initial conditions file](#infile_initial_conditions) (relative to `SETNGS_PATH`).
-
-1. `PARAMETER_TRIAL`: File path for the [trial parameters file](#infile_trial_parameters) (relative to `SETNGS_PATH`).
-
-1. `OUTPUT_PREFIX`: Text string prepended to each output filename to identify a specific model setup. Note that the user can further modify the output file name at run-time by using the `-s|--suffix` command-line option.
-
+The master configuration file is an [ASCII file](#infile_format_ASCII) and is provided to SUMMA at run-time as a command-line option. The path to this file needs to be supplied with the `-m` or `--master` command-line flag. The contents of this file orchestrate the remainder of the SUMMA run and are processed by the code in `build/source/hookup/summaFileManager.f90`. The file contents mostly consist of file paths that provide the actual information about the model configuration.  It also contains the run period and forcing time zone information.  
+
+The following items must be provided in the master configuration file. Order is not important, as the entries are each associated with a keyword.  Each keyword and entry pair must be on its own line, but may be followed by a comment (started by the '!' character), and you can add lines of comments between the items. Each entry must be enclosed in single quotes `'entry'`. The associations of the keywords to the actual variable name that is used in the SUMMA source code can be found in `summaFileManager.f90`, along with its default value where appropriate.
+
+`controlVersion`: Version of the file manager that should be used to process the master configuration file. At this time, this string should be equal to `'SUMMA_FILE_MANAGER_V2.0'`.  Note, this version of the code is not backward compatible with versions using `SUMMA_FILE_MANAGER_V1.0`.
```

Comment:
Change `SUMMA_FILE_MANAGER_V2.0` to `SUMMA_FILE_MANAGER_V3.0.0`

Reply:
Done